test/dataframe.py

```diff
@@ -179,3 +179,2519 @@ def test_float_dataframe():
     test_abs(ray_df, pandas_df)
     test_keys(ray_df, pandas_df)
     test_transpose(ray_df, pandas_df)
+
+
+def test_add():
+    pandas_df = pd.DataFrame({'col1': [0, 1, 2, 3],
```

Comment:
If this dataframe is the same in all of the tests, should we move it out as a global variable?

Alternatively, if we're worried about some tests mutating the global variable, we could make a function for creating the dataframe, e.g.,

```python
def create_test_dataframe():
    return pd.DataFrame({'col1': [0, 1, 2, 3],
                         'col2': [4, 5, 6, 7],
                         'col3': [8, 9, 10, 11],
                         'col4': [12, 13, 14, 15]})
```

And then in the tests

```python
pandas_df = create_test_dataframe()
```

What do you think?

Reply:
Yeah, I'll put it in a `create_test_dataframe()` function.  Since we're calling everything on the `ray_df`, I will just return that.